eclass/dlang-compilers.eclass

```diff
@@ -29,6 +29,7 @@ dlang-compilers_declare_versions() {
 		[7]="2.069 x86 amd64"
 		[8]="2.070 x86 amd64"
 		[9]="2.071 x86 amd64"
+		[10]="2.072 x86 amd64"
```

Comment:
That line should read "[10]="2.072 ~x86 ~amd64". It should match the KEYWORDS on the corresponding dmd ebuild.